akka-remote/src/main/mima-filters/2.5.x.backwards.excludes/_existing.excludes

```diff
@@ -74,3 +74,10 @@ ProblemFilters.exclude[DirectMissingMethodProblem]("akka.remote.artery.ArteryTra
 ProblemFilters.exclude[DirectMissingMethodProblem]("akka.remote.artery.ArteryTransport.createFlightRecorderEventSink")
 ProblemFilters.exclude[DirectMissingMethodProblem]("akka.remote.artery.ArteryTransport.createFlightRecorderEventSink$default$1")
 ProblemFilters.exclude[DirectMissingMethodProblem]("akka.remote.artery.ArteryTransport.flightRecorder")
+
+# Remove secure-cookie
+ProblemFilters.exclude[DirectMissingMethodProblem]("akka.remote.transport.AkkaProtocolSettings.RequireCookie")
+ProblemFilters.exclude[DirectMissingMethodProblem]("akka.remote.transport.AkkaProtocolSettings.SecureCookie")
+ProblemFilters.exclude[DirectMissingMethodProblem]("akka.remote.transport.HandshakeInfo.tupled")
+ProblemFilters.exclude[DirectMissingMethodProblem]("akka.remote.transport.HandshakeInfo.curried")
```

Comment:
Weird that those disappear when you add an explicit companion object, but seems harmless

Reply:
yeah, surprising, `HandshakeInfo` is one of those classes in classic remoting that probably shouldn't have been public in the first place so let's not care too much